src/components/CheckIn/AddPieceModal.js

```diff
@@ -31,6 +34,7 @@ const footer = (close, save) => (
     <Button
       buttonStyle="primary"
       data-test-add-piece-check-in
+      onClick={() => checkIn(getFormValues('AddPieceModalForm')(store.getState()))}
```

Comment:
IMHO, it's over-engineering. Simple `handleSubmit({isCheckIn: true})` should work. 
https://redux-form.com/7.4.2/docs/api/reduxform.md/#-code-onsubmit-function-code-optional-

Reply:
no answers, that's frustrating

Reply:
Apologize, tried this, but it didn't work or I did smth wrong

Reply:
plus it's not the best option to create a function in prop